README.md

```diff
@@ -329,6 +330,8 @@ It is essential to run (`source`) `activate learn-env` every time you start a ne
 
 However, there is one more step you need to perform. Firstly you need to ensure your terminal is running the learn-env virtual environment so you have the necessary packages. Then you need to go into your Jupyter Notebook and when viewing a notebook, click on "Kernel" in the top bar, then "Change Kernel" and then pick the learn-env kernel. You must make sure you're running the learn-env kernel whenever you're working in a Jupyter Notebook.
 
+![screen-jet](//Users/eli/Desktop/WeWork/Module1_DataExploration/dsc-1-01-05-setting-up-environment/learn_env.jpg)
+
```

Comment:
Hey @jet2016 I think you need to add this image to the repo and change the link to point to a local URL - learn_env.jpg